kedro-telemetry/kedro_telemetry/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Kedro plugin for collecting Kedro usage data."""
 
-__version__ = "0.3.0"
+__version__ = "0.5.0"
```

Comment:
No need to also bump `kedro-telemetry`here.

Reply:
Ah this one was an accident, I was on the wrong directory